internal/pkg/instance/instance.go

```diff
@@ -95,6 +99,29 @@ func getPath(privileged bool, username string) (string, error) {
 	return path, nil
 }
 
+func getDir(privileged bool, name string) (string, error) {
+	if err := CheckName(name); err != nil {
+		return "", err
+	}
+	path, err := getPath(privileged, "")
+	if err != nil {
+		return "", err
+	}
+	return filepath.Join(path, name), nil
+}
+
+// GetDirPrivileged returns directory where instances file will be stored
+// if instance is run with privileges
+func GetDirPrivileged(name string) (string, error) {
+	return getDir(true, name)
+}
+
+// GetDirUnprivileged returns directory where instances file will be stored
+// if instance is run without privileges
+func GetDirUnprivileged(name string) (string, error) {
```

Comment:
Is it used anywhere, or for the sake of symmetry here?

Reply:
May be required for user namespace code path

Reply:
got it